fapi-client/src/main/scala/com/gu/facia/api/models/curatedcontent.scala

```diff
@@ -61,7 +61,7 @@ object FaciaContent {
 
     CuratedContent(
       content,
-      trailMetaData.headline.orElse(contentFields.get("headline")),
```

Comment:
@maxharlow Thanks!

@adamnfish Maybe to make this concrete we should use `MetaData.headline` -> `fields.headline` -> `content.webTitle`?